src/data/sources.json

```diff
@@ -636,8 +594,8 @@
         "level1": {
           "allPopulationSex2006S": {
             "source": {
-              "href": "",
-              "title": ""
+                "href": "https://nigerianstat.gov.ng/resource/POPULATION%20PROJECTION%20Nigeria%20sgfn.xls",
```

Comment:
Can't we (or ask the data team) to add these to openAFRICA first and link to that page? Just trying to avoid linking directly to downloadable documents.